code/tests/checkpoint_example.cpp

```diff
@@ -221,8 +230,10 @@ class CheckpointExample : public BaseTestCase {
 
     if (bytes_read != strlen(TEST_TEXT)) {
       test_res->SetError(DataTestResult::kFileDataCorrupted);
+      test_res->error_description = check_file + ": read data size wrong";
     } else if (memcmp(TEST_TEXT, buf, strlen(TEST_TEXT)) != 0) {
       test_res->SetError(DataTestResult::kFileDataCorrupted);
+      test_res->error_description = check_file + ": read data incorrect";
```

Comment:
Similarly, lets print the index/prefix at which the buffers don't match

Reply:
done